### DISCUSSION TITLE
Disable Auto bed leveling before each print

### DISCUSSION THREAD

**Discussion by Avpman2** [xblax/flashforge_ad5m_klipper_mod#168] [Q&A]
How do I disable the auto bed leveling before each print? Is it in the FW or Orca? In the Orca-Flashforge build, there is a check box before sending the gcode to the printer to enable or disable auto bed leveling.

**Comment by b-fitzpatrick** [comment]
If there's not a mesh loaded, the start macro will run auto bed leveling. To prevent it, load a bed mesh (Tune page in Fluidd, never used Mainsail) prior to starting the print job. If there's no mesh to load, run auto_bed_level and save the configuration. The "auto" mesh will then be available next time you restart Klipper.

**Reply by Avpman2** [reply]
Excellent, that did it! Many thanks.
